conda-environment.yml

```diff
@@ -8,3 +8,4 @@ dependencies:
   - pip
   - poetry
   - nodejs
+  - redis
```

Comment:
Don't we rather need `redis-server` here?

Reply:
Hm, yes indeed. I thought it installed the server but it picked up my local server installation, so I did not notice